static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -60,7 +60,11 @@ define([
 
         // fromBotton:Boolean
         // will reverse the order of slots (this is useful for lazy loaded content)
-        fromBottom: false
+        fromBottom: false,
+
+        // considerScroll:Boolean
+        // will take scroll position into consideration (this is useful for liveblog ads)
+        considerScroll: false
```

Comment:
I added the `absoluteMinAbove` setting for this purpose, thinking we should measure from the top of the page (hence the `absolute`), and this is what we want to change. It is not used anywhere than in liveblogs, so you could actually use that setting instead of creating a new one.